hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -153,10 +157,13 @@ private void scan() {
             long now = Clock.currentTimeMillis();
             int backupTimeouts = 0;
             int invocationTimeouts = 0;
+            int invocationCount = 0;
```

Comment:
Ah, forgive me, it looked like a field of `InvocationMonitor`, not like a local variable. Makes total sense now.